docs: create a security policy (#2410)
FuelLabs/fuels-ts: SECURITY.md

> # Security Policy
> 
> The Fuel Typescript SDK is still in development, which means that our implementation may have issues.
> 
> We take security vulnerabilities very seriously. If you discover a security issue, please immediately bring it to our attention!
> 
> ## Reporting a Vulnerability
> 
> If you find a vulnerability or exploit, please [**report privately**](https://github.com/FuelLabs/fuels-ts/security/advisories/new).
> 
> Please **DO NOT file a public issue**.
> 
> We will send a response indicating the next steps to handle your report. The security team will let you know about the progress toward a fix and full announcement and may ask for additional information or guidance.
> 
> If the issue is an implementation weakness that cannot be immediately exploited or something that has not yet been deployed, just discuss it openly.
> 
> ## Reporting a non security bug
> 
> For non-security bugs, please simply file a GitHub [issue](https://github.com/FuelLabs/fuels-ts/issues/new).

Reviewer: I think we should be more direct, neutral, and broad in this communication.

<img width="570" alt="security" src="https://github.com/FuelLabs/fuels-ts/assets/26660/dd2aa6b4-a389-4ea2-8a81-80124639b6f3">

```suggestion
# Security Policy

The Fuel Typescript SDK is still in development, and we take security very seriously.

## Security Issues

If you find a vulnerability or exploit, please report it immediately and privately:
 - https://github.com/FuelLabs/fuels-ts/security/advisories/new

      > Please **DO NOT** file a public issue.

## Non-Security Issues

If the issue is not security-related, please report it publicly:
 - https://github.com/FuelLabs/fuels-ts/issues/new
```

# Security Policy

The Fuel Typescript SDK is still in it's development phase. This means that there may be issues in our implementation.

We take security vulnerabilities very seriously. If you discover a security issue,
please bring it to our attention right away!

## Reporting a Vulnerability

If you find a vulnerability or exploit please [**report privately**](https://github.com/FuelLabs/fuels-ts/security/advisories/new).

Please **DO NOT file a public issue**.

We will send a response indicating the next steps in handling your report. After the initial reply to your report, the security team will keep you informed of the progress towards a fix and full announcement, and may ask for additional information or guidance.

You can also report a vulnerability through the [npm contact form](https://www.npmjs.com/support) by selecting "I'm reporting a security vulnerability".

If the issue is an implementation weakness that cannot be immediately exploited or
something not yet deployed, just discuss it openly.

> If you need assistance, please reach out to [security@fuel.sh](mailto:security@fuel.sh).

Author: @arboleya the email exists as @petertonysmith94 says given we won't be getting these emails, I figure assistance that a someone may need in reporting, that we can't address, we can discuss internally and then respond appropriately. 5b51056b3
